h2o-py/h2o/explanation/_explain.py

```diff
@@ -1447,7 +1528,9 @@ def _handle_grouping(frame, grouping_column, save_plot_path, model, column, targ
                 binary_response_scale,
                 centered,
                 grouping_column=None,
-                **{'group_label':group_label}
+                output_graphing_data=output_graphing_data,
+                nbins=nbins,
+                **{'group_label':group_label, 'grouping_variable_value':curr_category}
```

Comment:
nit: This looks kinda fishy, wouldn't just using normal keyword arguments do the same?

Reply:
No, because we don't want to introduce new parameters `group_label` and `grouping_variable_value` into `ice_plot() `and `pd_plot()` methods - it would have no meaning for the user. When 'grouping_column' parameter is used, multiple ice plots are generated on subparts of input frame, grouped by the values of grouping column. So after the subparts are grouped, ice_plot/pd_plot is called on each of them. If I want to place an informative label about current group onto plot or into output table I need to pass this in somehow and with the current state of _explain.py this is the most straightforward way to do so but it shouldn't be a regular option of ice_plot/pd_plot()

Reply:
did without dictionary [a9dfa0d](https://github.com/h2oai/h2o-3/pull/6119/commits/a9dfa0dca78c37d998caa4dbd8d975cb361b9ea8)